packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json

```diff
@@ -2453,6 +2423,54 @@
         "refresh": 2,
         "skipUrlSync": false,
         "type": "interval"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "v4",
+          "value": "v4"
+        },
+        "datasource": "metrics",
+        "definition": "up",
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "tenant",
+        "options": [
+          {
+            "selected": true,
+            "text": "v4",
+            "value": "v4"
+          },
+          {
+            "selected": false,
+            "text": "system",
+            "value": "system"
+          },
+          {
+            "selected": false,
+            "text": "v3",
+            "value": "v3"
+          },
+          {
+            "selected": false,
+            "text": "launchpad",
+            "value": "launchpad"
+          }
+        ],
```

Comment:
I hope we can make this dynamic at some point! A drop-down, showing the actual tenants in the current cluster.

Reply:
This is a WIP.  The regex below does make this dynamic, but it saves the results here in the config.  I need to test omitting that part, or at least reverting back to "default".

Reply:
I think this is working now.  Will continue testing, and then update other dashboards.  Resolving this comment.